lib/livebook_cli/server.ex

```diff
@@ -53,24 +53,29 @@ defmodule LivebookCLI.Server do
     config_entries = opts_to_config(opts, [])
     put_config_entries(config_entries)
 
-    case start_server() do
-      :ok ->
-        if opts[:open] do
-          browser_open(LivebookWeb.Endpoint.access_url())
+    port = Application.get_env(:livebook, LivebookWeb.Endpoint)[:http][:port]
+    base_url = "http://localhost:#{port}"
+
+    case check_endpoint_availability(base_url) do
+      :livebook_running ->
+        IO.puts("Livebook already running on #{base_url}")
+        open_from_options(base_url, opts)
+
+      :taken ->
+        IO.puts(
```

Comment:
Should this be an error?

Reply:
Tried that initially, but then I didn't quite like this amount of red text:

![image](https://user-images.githubusercontent.com/17034772/131404480-da44a568-8f7f-4d50-b818-17c43c5e2efd.png)

Technically it's more of an actionable hint than error, wdyt?

Reply:
I would say it is an error because we could not fulfill the action (start the server). :)